omod/src/main/web/dashboardwidgets/dashboardwidgetscommons.service.js

```diff
@@ -127,4 +127,32 @@ export default class WidgetsCommons {
         return Boolean(obj.concept);
     }
 
+    /**
+     * Returns date translated according to preferred locale when Date Format displays the month as an abbreviation (MMM).
+     * Ex: Date: 02-Jan-2021 Format: DD-MMM-YYYY Locale: fr_FR --> Return: 02-janv.-2021
```

Comment:
I'm okay with writing this custom method, but I though moment would be able to handle this natively?

https://momentjs.com/docs/#/i18n/

Reply:
@mogoodrich I'll have to double check with @icrc-ifernandes as to why we decided for a custom function. The decision was taken a while back.

Reply:
Hi @mogoodrich . I know I've tried to use moment to handle this but for some reason I had some issues but I can't remember which ones. Let me try re-use it again to see what kind of issues I had and I'll come back to you asap. Thank you

Reply:
Cool... I don't want to be too much of a blocker on this, but would be good to note why we went with the approach we did, since there's potentially extra maintenance to maintain a custom approach.

Reply:
Hi @mogoodrich . After some changes on uiframework to have JS format date and the work from @mks-d to upgrade UIFR dep. to 3.21.0-SNAPSHOT on coreapps, I was able to implement formatDate with moment. It's now possible to translate widgets dates to locale language using moment. Let me know if you find any issues or you want to change any of the approaches. 
Thank you.